src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -441,38 +439,110 @@
+                         :sendType             send-type
+                         :addrFrom             from-address
+                         :addrTo               to-address
+                         :amountIn             amount-in
+                         :amountOut            amount-out
+                         :tokenID              token-id
+                         :toTokenID            to-token-id
+                         :disabledFromChainIDs disabled-from-chain-ids
+                         :disabledToChainIDs   disabled-to-chain-ids
+                         :gasFeeMode           gas-rates
+                         :fromLockedAmount     from-locked-amount}
+                        rest-keys)]]
      {:db            (cond-> db
                        :always (assoc-in [:wallet :ui :send :amount] amount)
                        :always (assoc-in [:wallet :ui :send :loading-suggested-routes?] true)
                        :always (assoc-in [:wallet :ui :send :sender-network-values]
                                 sender-network-values)
                        :always (assoc-in [:wallet :ui :send :receiver-network-values]
                                 receiver-network-values)
-                       :always (assoc-in [:wallet :ui :send :suggested-routes-call-timestamp] now)
                        :always (update-in [:wallet :ui :send] dissoc :network-links)
                        token   (assoc-in [:wallet :ui :send :token] token))
-      :json-rpc/call [{:method     "wallet_getSuggestedRoutes"
-                       :params     request-params
-                       :on-success (fn [suggested-routes]
-                                     (rf/dispatch [:wallet/suggested-routes-success suggested-routes
-                                                   now]))
-                       :on-error   (fn [error]
-                                     (rf/dispatch [:wallet/suggested-routes-error error])
-                                     (log/error "failed to get suggested routes"
-                                                {:event  :wallet/get-suggested-routes
-                                                 :error  error
-                                                 :params request-params}))}]})))
+      :json-rpc/call [{:method   "wallet_getSuggestedRoutesV2Async"
+                       :params   params
+                       :on-error (fn [error]
+                                   (rf/dispatch [:wallet/suggested-routes-error error])
+                                   (log/error "failed to get suggested routes (async)"
+                                              {:event  :wallet/get-suggested-routes
+                                               :error  error
+                                               :params params}))}]})))
+
+(rf/reg-event-fx :wallet/stop-get-suggested-routes
+ (fn []
+   {:json-rpc/call [{:method     "wallet_stopSuggestedRoutesV2AsyncCalcualtion"
```

Comment:
There's a typo in the word calculation.

Reply:
Yeah, the mistake is from how the API was named in status-go

https://github.com/status-im/status-go/blob/develop/services/wallet/api.go#L519-L523